samples/cpp/tutorial_code/ImgProc/HitMiss.cpp

```diff
@@ -15,18 +15,28 @@ int main(){
         0, 255, 0, 255, 0, 0, 255, 0,
         0, 255, 255, 255, 0, 0, 0, 0);
 
-    Mat kernel = (Mat_<uchar>(3, 3) <<
+    Mat kernel = (Mat_<int>(3, 3) << //can't be uchar because of -1
```

Comment:
Could you please remove this comment (to avoid users confusing about "unused" `uchar` - they don't see diffs in documentation).